### PR TITLE
Fix dependendencies and refresh README

### PR DESCRIPTION
**Pull request by nbttmbrg**
Removing references to vmware driver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,10 +33,10 @@ pip install molecule-gce
 
 If you plan on testing Windows instances you also need pywinrm:
 ```
-pip install pywinrm
+pip install pywinrm google-api-python-client oauth2client pycryptodome
```
Comment (ssbarnea) [inline README.md:36]: Just remove instruction about python dependencies, these should be part of package requirements (implicit).

Keep only `pip3 install "molecule-gce[windows]"` as we should have a windows extra that lists the windows specific requirements.

On the other hand I am wondering if we should not just include everything with normal driver, also windows requirements. Do you see a problem with that?

Comment (nbttmbrg) [reply README.md:36]: Ok, will delete all instructions about dependencies.

No problem with that

Comment (nbttmbrg) [reply README.md]: I applied the required changes

**Review by ssbarnea — APPROVED** [review]
While all the changes make sense please make them in separated PRs in order to have a good changelog.

**Pull request merged**